packages/remediations/src/steps/selectPlaybook.js

```diff
@@ -43,6 +50,8 @@ const SelectPlaybook = (props) => {
     const [ selectedPlaybook, setSelectedPlaybook ] = useState(values[EXISTING_PLAYBOOK]);
     const [ isLoadingRemediation, setIsLoadingRemediation ] = useState(false);
 
+    const { errors = [], warnings = [], resolutions = [], isLoading } = useSelector(({ resolutionsReducer }) => resolutionsReducer);
```

Comment:
This selector is too complex. I'd break it down to multiple consts and used `shallowEqual`

```JS
const errors = useSelector(({ resolutionsReducer }) => resolutionsReducer?.errors || [], shallowEqual);
const warnings = useSelector(({ resolutionsReducer }) => resolutionsReducer?.warnings || [], shallowEqual);
const resolutions = useSelector(({ resolutionsReducer }) => resolutionsReducer?.resolutions || [], shallowEqual);
const isLoading = useSelector(({ resolutionsReducer }) => resolutionsReducer?.isLoading);
```

I would also be carefull with the `equalityFn` as `shallowEqual`, for errors and warning that might be fine, but for `resolutions` it might be worth writing custom one.